backend/open_webui/main.py

```
@@ -1470,6 +1470,140 @@ async def chat_completed(
+            detail="Internal server error",
+        )
+
+
+# Get the signature of chat
+@app.get("/api/signature/{chat_completion_id}")
+async def chat_signature(
+    request: Request,
+    chat_completion_id: str,
+    model: str,
+    signing_algo: str = "ecdsa",
+    user=Depends(get_verified_user),
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)